simulator/simulator.go

```diff
@@ -0,0 +1,197 @@
+package simulator
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/env"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
+	sim "github.com/bitrise-io/go-xcode/simulator"
+)
+
+// InfoModel ...
+type InfoModel sim.InfoModel
```

Comment:
Should we drop the `Model` suffix since we are defining our own type (e.g. `Simulator` or `SimulatorInfo`)?

Reply:
@godrei Isn't `Info` too generic, even if it's in the `simulator` package? If it represents a simulator, shouldn't we call it `Simulator` or `SimulatorInfo`?

Reply:
Renamed to `simulator.Simulator` in https://github.com/bitrise-steplib/steps-xcode-test/pull/181/commits/36f27c80a3c7f1dab00405ded1cbda9bb684bb99